### PR TITLE
feat(services-auth-admin-api): Add default attributes and default grant types.

### PR DESCRIPTION
**Pull request by saevarma**
[Notion task](https://www.notion.so/Fill-out-default-attributes-based-on-client-type-c7ad3ef5240c4d6c80ca06095b70f3cf?pvs=4)

## What

Populating default client attributes on client creation based on the client type.
Also populating client grant type based on the client type.

## Why

To improve the client creation process.

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/auth-api-lib/src/lib/clients/admin/admin-clients.service.ts [hunk 1]
```diff
@@ -71,20 +75,74 @@ export class AdminClientsService {
...
+      updateAccessTokenClaimsOnRefresh: true,
+    }
+
     switch (clientType) {
+      case AdminClientType.web:
+        return baseAttributes
+      case AdminClientType.native:
+        return {
+          ...baseAttributes,
+          absoluteRefreshTokenLifetime: 30 * 24 * 60 * 60, // 30 days
```
Comment (eirikurn) [inline libs/auth-api-lib/src/lib/clients/admin/admin-clients.service.ts:107]: 1 year or disable?

File: libs/auth-api-lib/src/lib/clients/admin/admin-clients.service.ts [hunk 2]
```diff
@@ -71,20 +75,74 @@ export class AdminClientsService {
...
+          requireClientSecret: false,
+          slidingRefreshTokenLifetime: 7 * 24 * 60 * 60, // 7 days
```
Comment (eirikurn) [inline libs/auth-api-lib/src/lib/clients/admin/admin-clients.service.ts:109]: 3-6 months?

File: libs/auth-api-lib/src/lib/clients/admin/admin-clients.service.ts [hunk 3]
```diff
@@ -71,20 +75,74 @@ export class AdminClientsService {
...
       ...this.defaultClientAttributes(clientDto.clientType),
     })
+
+    await this.clientGrantType.create(this.defaultClientGrantTypes(client))
+
     return this.formatClient(client)
   }
 
-  defaultClientAttributes(clientType: AdminClientType) {
+  private defaultClientAttributes(clientType: AdminClientType) {
```
Comment (eirikurn) [inline libs/auth-api-lib/src/lib/clients/admin/admin-clients.service.ts:84]: Should have API tests for created clients of different types?

**Review by eirikurn — APPROVED** [review]

File: libs/auth-api-lib/src/lib/clients/admin/admin-clients.service.ts (hunk 3 above)
Comment (saevarma) [reply libs/auth-api-lib/src/lib/clients/admin/admin-clients.service.ts:84]: Ofc.. we even had the `it.todo` placeholders. Added tests, but for full implementation it requires changes from #10673 .

**Pull request merged**